scripts/code_generation/jsoncan/src/codegen/c_generation/c_config.py

```diff
@@ -51,6 +51,7 @@ class CTypesConfig(StrEnum):
     RX_TABLE_STRUCT = "{node}_RxMsgs"
     MSG_STRUCT = "{msg}_Signals"
     ALERT_ENUM = "{node}_{alert_type}Id"
+    CODE_ENUM = "{node}_{alert_type}Code"
```

Comment:
maybe pick a better name: code could refer to anything, maybe `AlertCode` or `AlertId`?

Reply:
also stay consistent with the rest of the PR, so pick either ID or code and use that everywhere.